apps/challenges/aws_utils.py

```diff
@@ -616,6 +616,112 @@ def scale_workers(queryset, num_of_tasks):
     return {"count": count, "failures": failures}
 
 
+def scale_resources(challenge, worker_cpu_cores, worker_memory):
+    """
+    The function called by scale_resources_by_challenge_pk to send the AWS ECS request to change the resources used by
```

Comment:
update to `The function called by scale_resources_by_challenge_pk to send the AWS ECS request to update the resources used by`